README.md

```diff
@@ -255,17 +252,16 @@ If you like this project...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+
```

Comment:
- [ ] [REQUIRED] As mentioned by the previous reviewer, please ensure that your MIT license link links to an actual license page. Currently, it leads to a 404 broken page. Please fix this by adding a file named `LICENSE.md` to your root folder. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE -https://choosealicense.com/licenses/mit/ 

**See the image below:** 
![image](https://user-images.githubusercontent.com/84156714/219359915-0ff34b82-8a3a-4034-8254-987118a868ff.png)

The following things are considered very important to make your project complete. I am only marking them as optional because the previous reviewer mistakenly omitted and did not mention them.

- [ ] [OPTIONAL] Your README Project title and descriptions are empty or void of meaningful content. Kindly add a proper title and description. 

    Your project README title description currently 👇🏾 

    ![image](https://user-images.githubusercontent.com/84156714/219360750-2c278aa9-99f0-45fb-99e0-e327fcb8fce1.png)

- [ ] The `Built-with section` is not well edited. You should remove the tech stack that has nothing to do with this project. Remove the REACT, EXPRESS and POSTGRES highlighted in the image below: 
![image](https://user-images.githubusercontent.com/84156714/219361266-bd6a8a7c-b418-4e84-ba64-75e7a3fa2e4b.png)